### DISCUSSION TITLE
`ctrl-e` keyboard shortcut is being hijacked on Mac (Safari and Chrome)

### DISCUSSION THREAD

**Discussion by ottumm** [community/community#6048] [Other Feature Feedback, Questions, & Ideas] [answered]
I use the keyboard shortcut `ctrl-e` all the time, to move the cursor to the end of a line. This is a system-wide shortcut that works in every well-behaved Mac app. Today, for the first time, I noticed that it's behavior in GitHub comment forms (issues, code reviews, discussions, etc) has changed to insert backticks around the current word.

I have years of muscle memory built up around this shortcut, as I'm sure many do, and I'm really hoping this is a bug. Any help or insight would be greatly appreciated!

**Comment by fdintino** [comment]
The tooltip on mac says "cmd + e", as do all of the other markdown shortcuts, but they're bound to both control and command. That seems like a bug to me.

**Reply by thomcc** [reply]
https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#styling-text indicates both command/control are intended to be supported for some of the hotkeys...

**Comment by thomcc** [comment]
This is true for `ctrl+k` too (normally deletes to end of line, now inserts markdown for a url)...

These emacs-style text editing and movement shortcuts have extremely deep muscle memory for me, and this makes the textboxes nearly unusable for me 😢.

**Comment by lastquestion** [comment]
This is really really annoying, and also, overrides the macos X default bindings that apply to every textbox and edit area in the OS.

I'm fine with adding new shortcuts for these things if it's configurable - I don't even need configurable shortcuts, just turn them off all the new ones.

**Comment by michellemerrill** [comment, marked as answer]
Thanks for bringing forward this feedback! Browser shortcuts that rely on `control` for Mac users should be functioning now.

**Reply by ottumm** [reply]
Excellent, thank you!!

**Reply by moble** [reply]
It still isn't working for me in Firefox.  I hard reloaded, cleared my cache, logged out and back in, restarted Firefox, tried it with "Disable Cache" in developer mode... I'm still getting backticks.  (I did try it in Brave, Chrome, and Safari, and those all worked as expected.)  Is there anything else I can try, or is this maybe still unresolved on Firefox?
